core/dev-tools/templates/simple/index.html

```diff
@@ -1,18 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Luigi Simple Dev</title>
-    <link rel="stylesheet" href="/public/luigi.css" />
-    <meta charset="utf-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1"
-    />
-  </head>
+<head>
+  <title>Luigi Simple Dev</title>
+  <link rel="stylesheet" href="/public/luigi.css" />
+  <meta charset="utf-8" />
+  <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1"
+  />
+</head>
 
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <script src="/public/luigi.js"></script>
-    <script src="/luigi-config.js"></script>
-  </body>
+<body>
+<noscript>You need to enable JavaScript to run this app.</noscript>
+<script src="/public/luigi.js"></script>
+<script src="/luigi-config.js"></script>
+</body>
+</html>
+</body>

```

Comment:
please remove this duplicated body and html tag.
I'm a bit confused because simpledev is listed in gitignore?

Reply:
ok